<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wavee</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="grid_layout">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="button_menu">
      <item>
       <widget class="QPushButton" name="refresh_button">
        <property name="text">
         <string>Refresh Graph</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="button_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="charge_widget" native="true">
      <layout class="QVBoxLayout" name="charge_layout">
       <item>
        <widget class="DraggableLabel" name="point_charge_circle">
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="point_charge_label">
         <property name="text">
          <string>Point Charge</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DraggableLabel" name="line_charge_drawing">
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="line_charge_label">
         <property name="text">
          <string>Infinite Line Charge</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DraggableLabel" name="circle_charge_drawing">
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="circle_charge_label">
         <property name="text">
          <string>Circle Charge</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DraggableLabel" name="ring_charge_drawing">
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ring_charge_label">
         <property name="text">
          <string>Ring Charge</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="DroppablePlotWidget" name="graph_widget" native="true"/>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="equation_layout">
      <item>
       <widget class="QWebEngineView" name="net_mag_equation_label" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWebEngineView" name="x_equation_label" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWebEngineView" name="y_equation_label" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DroppablePlotWidget</class>
   <extends>QWidget</extends>
   <header>view.droppable_plot_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DraggableLabel</class>
   <extends>QLabel</extends>
   <header>view.draggable_label</header>
  </customwidget>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>PyQt6.QtWebEngineWidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
